lua/git-conflict.lua

```diff
@@ -185,6 +185,8 @@ local function get_conflicted_files(dir, callback)
   job(cmd, function(data)
     local files = {}
     for _, filename in ipairs(data) do
+      local dir_sep = package.config:sub(1, 1)
```

Comment:
This doesn't need to be done every time this function is called this can be done in the scope of this file outside of any functions and saved to a file level variable and re-used here